packages/@react-aria/color/docs/useColorArea.mdx

```diff
@@ -216,35 +216,243 @@ function Example() {
+  position: relative;
+}
+
+.color-slider-track-background,
+.color-slider-track-color {
+  position: absolute;
+  height: 100%;
+}
+
+.color-slider-thumb {
+  position: absolute;
+  top: 14px;
+  border: 2px solid white;
+  box-shadow:
+    0 0 0 1px black,
+    inset 0 0 0 1px black;
+  border-radius: 50%;
+  box-sizing: border-box;
+  width: 20px;
+  height: 20px;
+}
+
+.color-slider-thumb.is-focused {
+  width: 32px;
+  height: 32px;
+}
+
+.color-slider-track-background,
+.color-slider-thumb-background {
+  background-size: 16px 16px;
+  background-position:
+    -2px -2px,
+    -2px 6px,
+    6px -10px,
+    -10px -2px;
+  background-color: white;
+  background-image:
+    linear-gradient(-45deg, transparent 75.5%, rgb(188, 188, 188) 75.5%),
+    linear-gradient(45deg, transparent 75.5%, rgb(188, 188, 188) 75.5%),
+    linear-gradient(-45deg, rgb(188, 188, 188) 25.5%, transparent 25.5%),
+    linear-gradient(45deg, rgb(188, 188, 188) 25.5%, transparent 25.5%);
+}
+
+.color-slider-thumb-background,
+.color-slider-thumb-color {
+  position: absolute;
+  border-radius: 50%;
+  width: 100%;
+  height: 100%;
+}
+```
+  </details>
+</details>
+
+### ColorSwatch
+
+The `ColorSwatch` component used in the example above implements an image preview of the color.
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show code</summary>
+
+```tsx example export=true render=false
+  function ColorSwatch (props) {
+    let {
+      value,
+      ...otherProps
+    } = props;
+    return (
+      <div
+        role="img"
+        className="color-preview-swatch"
+        aria-label={value.toString('css')}
+        {...otherProps}
+      >
+        <div className="color-preview-swatch-background" />
+        <div
+          className="color-preview-swatch-color"
+          style={{
+            backgroundColor: value.toString('css')
+          }} />
+      </div>
+    );
+  }
+```
+  <details>
+    <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show CSS</summary>
+
+```css
+  .color-preview-swatch {
+    display: inline-block;
+    border-radius: 4px;
+    forced-color-adjust: none;
+    position: relative;
+    width: 96px;
+    height: 96px;
+    overflow: hidden;
+  }
+
+  .color-preview-swatch-background {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    background-size: 16px 16px;
+    background-position:
+      -2px -2px,
+      -2px 6px,
+      6px -10px,
+      -10px -2px;
+    background-color: white;
+    background-image:
+      linear-gradient(-45deg, transparent 75.5%, rgb(188, 188, 188) 75.5%),
+      linear-gradient(45deg, transparent 75.5%, rgb(188, 188, 188) 75.5%),
+      linear-gradient(-45deg, rgb(188, 188, 188) 25.5%, transparent 25.5%),
+      linear-gradient(45deg, rgb(188, 188, 188) 25.5%, transparent 25.5%);
+  }
+
+  .color-preview-swatch-color {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+  }
+```
+  </details>
+</details>
+
+
+
 
 ### onChangeEnd
```

Comment:
Since they are very similar, I still think this example could potentially be combined with the "Controlled" one if you had two text labels at the bottom showing the current controlled value and also the ending value (like the "Events" section in RSP docs).